### ISSUE TITLE
RethinkDB sink does not use SSL configuration

### ISSUE THREAD

**Issue by musflood** [lensesio/stream-reactor#674]
### What version of the Stream Reactor are you reporting this issue for?

1.2.3

### Are you running the correct version of Kafka/Confluent for the Stream reactor release?

Yes, running the RethinkDB Sink with Confluent 5.4.1

### Do you have a supported version of the data source/sink .i.e Cassandra 3.0.9?

Using RethinkDB version 2.3.6

### Have you read the [docs](https://docs.lenses.io/connectors/)?

Yes

### What is the expected behaviour?

Supplying the SSL cert file and key should allow connection to a secure RethinkDB proxy server.

### What was observed?

The connector worked fine with a local RethinkDB instance, which had no authentication or SSL in place. However, when I attempt to connect to a remote RethinkDB proxy server, which requires SSL, I consistently get a `com.rethinkdb.gen.exc.ReqlDriverError: Connection timed out.` error.

On inspection of `ReThinkSinkConnector.initializeTables`, discovered that the initialization of the connector makes a connection to RethinkDB in order to set up the tables, however that connection is made using only the host and port.
```scala
val conn = rethink.connection().hostname(rethinkHost).port(port).connect()
```
Which explains why the connection is timing out at that point when the host is protected with SSL.

Suspect that this line would also cause issues when attempting to connect to a database protected with a username and password as well.

Side note: RethinkDB allows connecting using a SSL cert and a username/password, so not sure why that is not an option for the connector?

### What is your connector properties configuration (my-connector.properties)?

```
connect.progress.enabled = false
connect.rethink.db = asset_data
connect.rethink.error.policy = THROW
connect.rethink.host = [hidden]
connect.rethink.kcql = UPSERT INTO kafka_asset_positions SELECT * FROM dc_1_asset_positions AUTOCREATE PK asset_id
connect.rethink.max.retries = 20
connect.rethink.password = [hidden]
connect.rethink.port = [hidden]
connect.rethink.rethink.auth.key = [hidden]
connect.rethink.rethink.cert.file = /usr/share/creds/cacert
connect.rethink.rethink.username =
connect.rethink.retry.interval = 60000
```

### Please provide full log files (redact and sensitive information)

```
com.rethinkdb.gen.exc.ReqlDriverError: Connection timed out.
	at com.rethinkdb.net.SocketWrapper.connect(SocketWrapper.java:91)
	at com.rethinkdb.net.Connection.connect(Connection.java:100)
	at com.rethinkdb.net.Connection.reconnect(Connection.java:94)
	at com.rethinkdb.net.Connection.reconnect(Connection.java:83)
	at com.rethinkdb.net.Connection$Builder.connect(Connection.java:422)
	at com.datamountaineer.streamreactor.connect.rethink.sink.ReThinkSinkConnector.initializeTables(ReThinkSinkConnector.scala:83)
	at com.datamountaineer.streamreactor.connect.rethink.sink.ReThinkSinkConnector.start(ReThinkSinkConnector.scala:73)
	at org.apache.kafka.connect.runtime.WorkerConnector.doStart(WorkerConnector.java:111)
	at org.apache.kafka.connect.runtime.WorkerConnector.start(WorkerConnector.java:136)
	at org.apache.kafka.connect.runtime.WorkerConnector.transitionTo(WorkerConnector.java:196)
	at org.apache.kafka.connect.runtime.Worker.startConnector(Worker.java:260)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.startConnector(DistributedHerder.java:1183)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.access$1400(DistributedHerder.java:125)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$15.call(DistributedHerder.java:1199)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$15.call(DistributedHerder.java:1195)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readByte(DataInputStream.java:267)
	at com.rethinkdb.net.SocketWrapper.readNullTerminatedString(SocketWrapper.java:117)
	at com.rethinkdb.net.SocketWrapper.connect(SocketWrapper.java:87)
	... 18 more
```

**Comment by andrewstevenson** [comment]
@musflood The driver sets the connection to use the cert file provided so I'm not sure why you think its not?

https://github.com/lensesio/stream-reactor/blob/1.2.3/kafka-connect-rethink/src/main/scala/com/datamountaineer/streamreactor/connect/rethink/ReThinkConnection.scala#L62

**Comment by musflood** [comment]
@andrewstevenson Yes, the driver does use the cert file in `ReThinkConnection`, however when the connector is starting up, it uses `RethinkDB.r.connection()` directly and does not use the cert file to make the connection.

**Comment by andrewstevenson** [comment]
Fixed in #669
